libs/react-components/src/lib/OrganizationBox/OrganizationBox.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import { render } from "@testing-library/react";
+
+import OrganizationBox from "./OrganizationBox";
+
+describe("OrganizationBox", () => {
+  it("should render successfully", () => {
```

Comment:
Throws `TypeError: Cannot read property 'src' of undefined`. Didn't the tests ran before push automatically?

Reply:
Ok…could you please comment out the content of the test (don't remove it) and add a TODO node to write tests?

Reply:
This will throw an `Your test suite must contain at least one test.` error. Please add
```suggestion
describe("OrganizationBox", () => {
  it("should render successfully", () => {
    // TODO: write reasonable tests
  });
});
```

Reply:
Comment everything out throws an `Your test suite must contain at least one test.` error. Please replace the file content with
```javascript
// TODO: write reasonable tests
test("renders successfully", () => {});
```
Maybe we should change the Nx templates.

Reply:
done